src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentBase.java

```diff
@@ -71,10 +68,9 @@ public class SSHAgentBase {
             "-----END OPENSSH PRIVATE KEY-----";
 
     protected void startMockSSHServer() throws Exception {
-        File hostKey = new File(System.getProperty("java.io.tmpdir") + "/key.ser");
-        hostKey.delete(); // do not carry from test to test
```

Comment:
because this was using the same file accross tests and deleting it if it existed!
(running multiple tests in parallel!)